ksql-cli/src/main/java/io/confluent/ksql/cli/console/Console.java

```diff
@@ -475,18 +476,47 @@ private static String getTopicReplicaInfo(List<Integer> replicaSizes) {
     }
   }
 
-  private String formatFieldType(FieldSchemaInfo field, String keyField) {
+  private String schemaToTypeString(SchemaInfo schema) {
+    // For now just dump the whole type out into 1 string.
+    // In the future we should consider a more readable format
+    switch (schema.getType()) {
+      case ARRAY:
+        return new StringBuilder()
+            .append(SchemaInfo.Type.ARRAY.name() + "<")
+            .append(schemaToTypeString(schema.getMemberSchema()))
+            .append(">")
+            .toString();
+      case MAP:
+        return new StringBuilder()
+            .append(SchemaInfo.Type.MAP.name())
+            .append("<" + SchemaInfo.Type.STRING + ", ")
+            .append(schemaToTypeString(schema.getMemberSchema()))
+            .append(">")
+            .toString();
+      case STRUCT:
+        return schema.getFields()
+            .stream()
+            .map(f -> f.getName() + " " + schemaToTypeString(f.getSchema()))
+            .collect(Collectors.joining(", ", SchemaInfo.Type.STRUCT.name() + "<", ">"));
+      case STRING:
+        return "VARCHAR(STRING)";
```

Comment:
in my usage of ksql, this is not a valid type that I can send to register a field

Reply:
Its just what we print from the CLI, not present in the response itself. I don't really have a strong preference either way. Maybe someone with more ux gravitas can chime in?
@miguno - any preference on how we show string schemas in the cli?

```
ksql> describe test;
...
 FOO    | VARCHAR(STRING)
...
```
vs
```
ksql> describe test;
...
 FOO    | VARCHAR
...
```
vs
```
ksql> describe test;
...
 FOO    | STRING
...
```